www-servers/nginx/files/nginx-r2.service

```diff
@@ -4,7 +4,8 @@ After=network.target remote-fs.target nss-lookup.target
 
 [Service]
 Type=forking
-PIDFile=/run/nginx.pid
+PIDFile=/run/nginx/nginx.pid
```

Comment:
That patch is not enough, we would have to patch the eclass:
https://github.com/gentoo/gentoo/blob/2ea1f8c5e1a10b185ff60eaded354411e63c129f/eclass/nginx.eclass#L594